specs/mediawiki/v1/content.yaml

```diff
@@ -75,7 +70,7 @@ paths:
         # See also http://tools.ietf.org/html/rfc7231#section-7.1.2
         uri: /{domain}/sys/parsoid/html/{title}/latest
 
-  /{module:page}/{title}/html/:
+  /{module:page}/html/{title}/:
```

Comment:
I think `Accept` is a great way to request the same semantic information in different formats, but am not convinced that it is the right way to return different resources.

Swagger lets us document each possible return content-type.